openstack/nova/console/shellinabox/revproxy.lua

```diff
@@ -3,7 +3,7 @@ local _M = {}
 local token_lookup = require "token_lookup"
 local get_token = token_lookup.get_token
 local lookup = token_lookup.lookup
-local host_from_entry = token_lookup.token_lookup.lookup
+local host_from_entry = token_lookup.fix_nova_shellinabox_console
```

Comment:
Shouldn't this be `token_lookup.host_from_entry`?

Reply:
Sorry you are right

Reply:
This is still not correct. It now points to `token_lookup.lookup` instead of `token_lookup.host_from_entry`.

Reply:
You're right... maybe I better lie down. Sorry for the extra work.